client/components/tinymce/plugins/contact-form/plugin.js

```diff
@@ -23,26 +26,38 @@ const contactForm = editor => {
 	} );
 
 	editor.on( 'remove', () => {
-		ReactDom.unmountComponentAtNode( node );
+		unmountComponentAtNode( node );
 		node.parentNode.removeChild( node );
 		node = null;
 	} );
 
-	editor.addCommand( 'WP_ContactForm', () => {
+	editor.addCommand( 'wpcomContactForm', content => {
```

Comment:
What is `content` here? If this is never set, should this be the reducer default and not require an initialization action?